Sources/AppStoreConnectCLI/Commands/Certificates/ListCertificatesCommand.swift

```diff
@@ -55,7 +55,10 @@ struct ListCertificatesCommand: CommonParsableCommand {
 
                 let file = try certificateProcessor.write($0)
 
-                print("📥 Certificate '\($0.name ?? "")' downloaded to: \(file.path)")
+                // Command output is parsable by default. Only print if user is enabling verbosity or output is a `.table`
+                if common.verbose || common.outputFormat == .table {
```

Comment:
We seem to have to use `common.verbose || common.outputFormat == .table` quite a bit here.
What do you think about making `verbose` be `Bool?` (defaulting to `nil`) and then we can add a computed var either a Bool or enum to check.

Thinking along these lines:

```swift
// CommonParsableCommand.swift

enum PrintLevel {
  case quiet
  case verbose
}

// CommonOptions

var printLevel: PrintLevel {
  switch verbose {
    case true:
      return .verbose
    case false:
      return .quiet
    case nil:
      return common.outputFormat == .table ? .verbose : .quiet
  }
}

// Usage

if common.printLevel == .verbose {
  ...
}
```

Reply:
Overall, this is a good suggestion and I'll make these changes with small tweaks. I'll leave `verbose` as a `Bool` and I'll resolve the `common.outputFormat == .table` ternary in the `switch` statement itself. Should look something like this:
```swift
// CommonOptions

var printLevel: PrintLevel {
  switch (verbose, outputFormat == .table) {
    case (true, _), (_, true):
      return .verbose
    case (false, false):
      return .quiet
  }
}
```

Reply:
Sounds good, thanks @csjones